packages/playwright-core/src/server/bidi/bidiInput.ts

```diff
@@ -77,8 +77,8 @@ export class RawMouseImpl implements input.RawMouse {
 
   async move(x: number, y: number, button: types.MouseButton | 'none', buttons: Set<types.MouseButton>, modifiers: Set<types.KeyboardModifier>, forClick: boolean): Promise<void> {
     // Bidi throws when x/y are not integers.
-    x = Math.round(x);
-    y = Math.round(y);
+    x = Math.floor(x);
```

Comment:
There is already a block that fixes the same problem for Firefox, accounting for all kinds of oddly shaped elements: https://github.com/microsoft/playwright/blob/main/packages/playwright-core/src/server/dom.ts#L269. Perhaps we should generalize that check through `PageDelegate.shouldClickAtIntegerCoordinates()`?

Reply:
This referenced command in the DOM module is using Content Quads to calculate the position. This is a feature that is not yet supported in WebDriver BiDi. So I feel it might be inappropriate to be used right now. If you agree I could at least add a comment to my changes to reference that other method for future adaption once quads are supported.

Reply:
Even though content quads are not supported in BiDi yet, there is a polyfill in `BidiPage` for it. Otherwise, click won't work in Playwright at all. Therefore, we can still assume there are *some* content quads available and perform smart integer rounding, wdyt?

Reply:
Searching for an integer point in a quad is a separate issue. This PR is good per se as it aligns the logic with what we have in `ffInput.ts` where we use `floor` rather than `round`. Let's land it.